cmd/utils/flaggroup.go

```diff
@@ -288,7 +288,7 @@ var FlagGroups = []FlagGroup{
 			VTRecoveryIntervalFlag,
 			ServiceChainAnchoringFlag,
 			ServiceChainNewAccountFlag,
-			ServiceChainDefaultTxGasLimitFlag,
+			ServiceChainParentOperatorDefaultTxGasLimitFlag,
```

Comment:
Isn't ServiceChainChildOperatorDefaultTxGasLimitFlag should be here too?

Reply:
Oh I missed it. Thanks for your review.